.github/workflows/groovy_checkstyle.yml

```diff
@@ -4,6 +4,7 @@ on: [push]
 
 jobs:
   build:
+    name: checkstyle
```

Comment:
wait, you need to update the 'build' one line above, right?

Reply:
No this works too 😄

Reply:
I think I did not need to change this actions name but I did it anyways :)

Reply:
but it does not make sense to me to call it "build" in the line above but give it another name afterwards ... more code where not necessary.